### DISCUSSION TITLE
Can I run the script on Alpine 3.13 again without losing the configuration of NRM?

### DISCUSSION THREAD

**Discussion by witzker** [ej52/proxmox-scripts#18] [Q&A] [answered]
Hi I have updated Alpine from 3.12 to 3.13
now I have lost the login screen URL:81

Also, pls. see here:
https://github.com/ej52/proxmox-scripts/issues/16

Will I lose my configuration when executing the installation script on Alpine again?

**Comment by ej52** [comment, marked as answer]
@witzker Sorry didn't realize there was a discussion about that PR :/

You will not lose your config when you run `setup.sh`, only the dependencies are updated.
When running `create.sh` on proxmox, it will create a brand new CT. So in that case none of your data will be there.
